core/domain/stats_domain.py

```diff
@@ -584,13 +644,27 @@ def __init__(self, answer, frequency):
         self.frequency = frequency
 
     def to_raw_type(self):
+        """Returns dict for specific answer in its raw type.
+
+        Returns:
+            dict(str, str). The specific answer dict.
```

Comment:
This is not a dict(str, str). Enumerate the keys and values properly.

Reply:
Done! The values are of type both 'str' and 'int'. So I've added dict(str, *) here and enumerated the key-values pairs.